src/main/java/org/sonarlint/intellij/ui/HotspotCellRenderer.java

```diff
@@ -36,16 +36,14 @@ class HotspotCellRenderer implements ListCellRenderer {
   private static final int WHITE = 0xffffff;
   private final JBColor whiteForeground = new JBColor(WHITE, WHITE);
 
-  private static final String ICON_KEY = "security_hotspot_";
-
   @Override
   public Component getListCellRendererComponent(JList list, Object value, int index, boolean isSelected, boolean cellHasFocus) {
     var hotspot = (LiveSecurityHotspot) value;
     var layout = new FlowLayout(FlowLayout.LEFT);
     var panel = new JPanel(layout);
 
     var shIconLabel = new JBLabel();
-    var icon = SonarLintIcons.type(ICON_KEY + hotspot.getVulnerabilityProbability().name());
+    var icon = SonarLintIcons.hotspotTypeWithProbability(hotspot.getVulnerabilityProbability());
```

Comment:
I think this file is not used anymore and can be deleted

Reply:
Good catch!